tests/reject-tests.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+
+# Copyright (C) 2014 Nikos Mavrogiannopoulos
+#
+# License: BSD
+
+srcdir="${srcdir:-.}"
+
+echo "***********************************************"
+echo "This test will use a radius server on localhost"
+echo "and which can be executed with run-server.sh.  "
+echo "Test passes invalid client credentials and     "
+echo "expects it to be rejected by Server.           "
+echo "***********************************************"
+
+TMPFILE=tmp$$.out
+
+if test -z "$SERVER_IP";then
+	echo "the variable SERVER_IP is not defined"
+	exit 77
+fi
+
+PID=$$
+sed -e 's/localhost/'$SERVER_IP'/g' -e 's/servers-temp/servers-temp'$PID'/g' <$srcdir/radiusclient.conf >radiusclient-temp$PID.conf
+sed 's/localhost/'$SERVER_IP'/g' <$srcdir/servers >servers-temp$PID
+
```

Comment:
Testing a REJECT in general and not specific to EAP.  Usually, the REJECT message may contain any kind of payload and EAP is one such kind. The changes made in this pull request allow all kinds of AVP-list pairs (EAP, ..) to be copied back to caller. 

And tests here pass invalid credentials to RADIUS Server and verify whether the configs received do not belong to the set of user with valid credentials.